plugins/email/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/email/rest/EMailResource.java

```diff
@@ -50,6 +50,18 @@
 	@Reference
 	private EMailLogic logic;
 
+	/**
+	 * Get the currently saved mail credentials
+	 * 
+	 * @return mail credentials
+	 */
+	@GET
+	@Path("getCredentials")
+	@Produces(MediaType.APPLICATION_JSON)
+	public EMailCredentials getCredentials() {
```

Comment:
remove this method. for security reasons don't make it possible to read the password